client/cli/src/client_proxy.rs

```diff
@@ -748,8 +745,8 @@ impl ClientProxy {
     pub fn submit_signed_transaction(
         &mut self,
         raw_txn: RawTransaction,
-        public_key: Ed25519PublicKey,
-        signature: Ed25519Signature,
+        public_key: ed25519::VerifyingKey,
```

Comment:
IMHO we should not break the consistency between variables and types. So, I would go for `public_key: XXXPublicKey` or `verifying_key: XXXVerifyingKey`

Reply:
actually, in network we do call it a signing key:

```
    pub signing_keys: KeyPair<Ed25519PrivateKey>,
```

Reply:
(even signing "keys" :D)